enterprise/models.py

```diff
@@ -1101,16 +1101,6 @@ class Meta:
         verbose_name_plural = _("Branding Configurations")
         ordering = ['created']
 
-    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
-        """Save the enterprise customer branding config."""
-        if self.pk is None:
-            logo_image = self.logo
-            self.logo = None
-            super(EnterpriseCustomerBrandingConfiguration, self).save(*args, **kwargs)
-            self.logo = logo_image
-
-        super(EnterpriseCustomerBrandingConfiguration, self).save(*args, **kwargs)
-
```

Comment:
this override method calls save() method twice in case we do not pass it while using its Factory.
and causing an Integrity Error.
And I also can't find any good use of this override method so I removed it and it didn't fail any model test too.

Reply:
This is a bizarre piece of code that certainly warranted a comment explaining its existence. If the id isn't set, then we clear the logo field, save, then reset the logo field and save again? There must have been a reason for this. I agree with removing it, but we should pay close attention to this model after merging.